operations/compare-helm-with-jsonnet/helm/06-pods/kustomization.yaml

```diff
@@ -0,0 +1,81 @@
+resources:
+  - ../05-services
+
+components:
+  - ../../components/pods
+
+# Jsonnet doesn't use a config file, do we can delete the whole object
+patches:
+  - target:
+      name: mimir-config
+    patch: |-
+      $patch: delete
+      apiVersion: v1
+      kind: Secret
+      metadata:
+        name: mimir-config
+
+  # TODO(logiraptor): Helm adds a grpc port to the overrides exporter, Jsonnet does not.
+  - target:
+      name: mimir-overrides-exporter 
+      kind: Deployment
+    patch: |-
+      - op: remove
+        path: /spec/template/spec/containers/0/ports/1
+
+  # TODO(logiraptor): Helm adds a gossip port to the ruler, Jsonnet does not. This seems like a bug on the Jsonnet side, since all other ring-connected components do have a gossip port.
+  - target:
+      name: mimir-ruler
+      kind: Deployment
+    patch: |-
+      - op: remove
+        path: /spec/template/spec/containers/0/ports/2
+
+  # TODO(logiraptor): Helm adds an emptyDir volume to several components. This is likely due to the fact that readOnlyRootFilesystem=true and some ephemeral data is written
+  - target:
+      name: mimir-(distributor|query-frontend|querier|ruler|querier|overrides-exporter|alertmanager)
+      kind: Deployment|StatefulSet
+    patch: |-
+      - op: remove
+        path: /spec/template/spec/volumes/1
+
+  # Helm attempts to automatically discover which PDB apiVersion is supported,
```

Comment:
FYI, In the `ct` tests we circumvent this "flakyness" by setting the kubernetes version explicitly: 

https://github.com/grafana/mimir/blob/23a9fe46b53fc30ab8e3fdecf224c7033d04c2fb/operations/helm/charts/mimir-distributed/ci/test-oss-values.yaml#L2